drivers/ethernet/eth_native_posix_adapt.c

```diff
@@ -95,7 +95,7 @@ static int ssystem(const char *fmt, ...)
 	int ret;
 
 	va_start(ap, fmt);
-	vsnprintf(cmd, sizeof(cmd), fmt, ap);
+	vsnprintk(cmd, sizeof(cmd), fmt, ap);
```

Comment:
Nack to this specific change. The `vsnprintf()` is actually a call to Linux libc here.

Reply:
Removed!